io/src/main/scala/sbt/io/syntax.scala

```diff
@@ -1,11 +1,21 @@
 package sbt.io
+import java.io.File
+
+import sbt.io.Glob.GlobBuilder
+import sbt.io.PathFinderCombinator.SingleFilePathFinderCombinator
 
 private[sbt] trait Alternative[A, B] {
   def |(g: A => Option[B]): A => Option[B]
 }
 
-sealed abstract class IOSyntax1 {
-  implicit def singleFileFinder(file: java.io.File): PathFinder = PathFinder(file)
+sealed trait IOSyntax2 {
+  implicit def lowPriorityFileFinder(file: File): PathFinder = PathFinder(file)
+}
+sealed abstract class IOSyntax1 extends IOSyntax2 {
+  def singleFileFinder(file: File): PathFinder = PathFinder(file)
```

Comment:
Could you deprecate this so we can find this later when we go sbt 2?
And also could you comment the fact that we are leaving this around for bincompat reason, but we are migrating the immediate semantics of Path-like behavior to `GlobBuilder` starting 1.3?

Reply:
Definitely. This was an oversight.